src/Development/Shake/Internal/Derived.hs

```diff
@@ -72,22 +75,30 @@ addShakeExtra x = Map.insert (typeOf x) (toDyn x)
 
 -- | @copyFile' old new@ copies the existing file from @old@ to @new@.
 --   The @old@ file will be tracked as a dependency.
+--   Also creates the new directory if necessary.
 copyFile' :: FilePath -> FilePath -> Action ()
 copyFile' old new = do
     need [old]
     putLoud $ "Copying from " ++ old ++ " to " ++ new
+    createDirIfMissing new
```

Comment:
Maybe I'm wrong (and I usually am), but given `copyFile' "a" "b"`, wouldn't this end up creating the directory `b` and then copying `a` into it, resulting in `b/a` instead of `b`?